css/video.scss

```
@@ -473,3 +473,7 @@ video {
 
 	opacity: 0.7;
 }
+
+#muteWrapper .icon-audio-off + .volume-indicator {
```

Since this is SCSS you could move this into the above brackets and then use:
```suggestion
&+.icon-audio-off {
```